openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -733,6 +753,46 @@ def _apply_cdn_to_url(self, url, base_url):
 
         return urlunparse(('', base_url, path, params, query, fragment))
 
+    @cached_property
+    def _original_course(self):
```

Comment:
The intention is to get all of the required information from the `CourseOverview`, without having to hit the modulestore. But some of the machinery for instantiating tabs checks other attributes on the course. We could decide whether to add these to the overview, or proxy to the course object.